releasenotes/notes/collect-manifest-423cb6118ed549b0.yaml

```diff
@@ -8,4 +8,4 @@
 ---
 features:
   - |
-    Collect manifests.
+    Collect manifests (pods yaml) in process check.
```

Comment:
I think we only need one release note (the agent one here)? But not 100% sure. Can't remember that we ever created 2 in a row

Reply:
I might be wrong though so lets sync on that one

Reply:
As we discussed, I deleted releasenotes